xpresso/responses.py

```diff
@@ -33,11 +33,11 @@
 def get_response(request: Request) -> Response:
     try:
         return request.scope["extensions"]["xpresso"]["response"]  # type: ignore[return-type]
-    except KeyError:
+    except KeyError as e:
         raise LookupError(
             "xpresso.responses.get_response was called"
             " before the endpoint has finished executing or the endpoint raised an exception"
-        )
+        ) from e
```

Comment:
I think this is one where we're better off doing `from None`: the fact that we got a KeyError is an internal implementation detail that adds no useful information for the user, it just clutters their traceback.